config/conf.toml

```diff
@@ -2,3 +2,11 @@
 tsdb.backend = 'marketstore'
 tsdb.host = 'localhost'
 tsdb.grpc_port = 5995
+
+[ui]
+# set custom font + size which will scale entire UI
+# font_size = 16
+# font_name = 'Monospaced'
```

Comment:
@ebisu4 tellin me there's issues getting this to work on windows @sirdinkus, so if you could test it a bit it'd be handy 😉